### PR TITLE
Photo Component Added

### PR DESCRIPTION
**Pull request by isaack87**

### CODE REVIEW

**Comment by isaack87** [comment]
forgot to add something

**Comment by isaack87** [comment]
Created a Provider Component which will allow me to send props to any stateless component without the use of (PROP drilling) for much cleaner and nicer code.

File: client/dist/styles.css [hunk 1]
```diff
@@ -189,42 +179,37 @@
...
   border: .3em dotted orange;
   flex-direction: row;
-  margin: -1em;
 }
 .answerbox {
-  align-items: center;
-  justify-content: center;
-  display: block;
-  width: 59em;
+  display: flex;
```
Comment (heltan) [inline client/dist/styles.css:203]: good job on use of the flex box, it looks good

File: client/src/components/productState.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React, { useState, createContext} from 'react';
```
Comment (heltan) [inline client/src/components/productState.jsx]: nice use of usestate! haven't heard of this before but it looks useful for passing state

**Review by heltan — APPROVED** [review]

**Pull request merged**